packages/devtools/package.json

```diff
@@ -35,7 +35,8 @@
     "@wdio/protocols": "6.0.0-alpha.1",
     "@wdio/utils": "6.0.0-alpha.1",
     "chrome-launcher": "^0.11.1",
-    "puppeteer-core": "^1.18.1"
+    "puppeteer-core": "^2.1.1",
+    "puppeteer-firefox": "^0.5.0"
```

Comment:
We have this already in `optionalDependencies` so this should be removed.
```suggestion
```

Reply:
OK I will need to fix the unit tests for that , too, let me see what I can do.